draft-opsarea-rfc5706bis.md

```diff
@@ -1153,6 +1158,65 @@ Information Models and Data Models
    Protocol designers should consider both data-oriented and task-
    oriented authority levels and policy.
 
+## Tooling Considerations {#sec-tooling}
+
```

Comment:
Need to define tools.

"Tools refers to software systems or utilities used by network operators to deploy, configure, monitor, troubleshoot, and manage networks or network protocols in real-world operational environments."

for example.